setup.cfg

```diff
@@ -78,6 +78,7 @@ tests =
   pytest-intercept-remote>=1.2
   pytest-mock
   pytest-mpl>=0.12 # First version to support our figure tests
+  reproject
```

Comment:
Should we add reproject as a map dependency at this point? If not should we add it's own extra? i.e. do we think that `install sunpy[all]` should bring in reproject? reproject [has minimal requirements](https://github.com/astropy/reproject/blob/master/setup.cfg#L19-L23) (healpix has no requirements tho is all C) though it is a bunch of C/Cython to compile.

ndcube [added it as an optional](https://github.com/sunpy/ndcube/blob/main/setup.cfg#L54)

Reply:
<shrug>

Reply:
After discussing this at the community meeting today, I am pro adding reproject to the map extra.

Reply:
Done